src/main/Embeddable.tsx

```diff
@@ -2,43 +2,46 @@ import * as React from 'react'
 import {didPropsChange} from './util'
 import { ChartDesigner, CommonConfigOptions, EventManager, Region, SeatingChart, Seatsio } from '@seatsio/seatsio-types'
 
-export type EmbeddableProps<T> = {
-    onRenderStarted?: (chart: SeatingChart) => void
+export type EmbeddableProps<T > = {
+    onRenderStarted?: (chart: SeatingChart | EventManager) => void
     chartJsUrl?: string
     region: Region
 } & T
 
 export default abstract class Embeddable<T extends CommonConfigOptions> extends React.Component<EmbeddableProps<T>> {
     private container: React.RefObject<HTMLDivElement>
-    private rendering?: Promise<void>
     private chart: SeatingChart
 
+    private static seatsioBundles: { [key: string]: Promise<Seatsio> } = {}
```

Comment:
I think we need to be careful with a static variable here.

Suppose our customer renders a chart, hides it, and then renders it again. Upon rerendering, `componentDidMount` gets invoked. But because `Embeddable.seatsioBundles` already has a value, we don't rerender the chart, so nothing gets shown.

I added a dropdown to our playground to illustrate what I mean. It works fine with the old `Embeddable` code, but not with the new code.

Reply:
Added a new property `firstRender` that is part of the check when component mounts. Hiding and showing the component causes the constructor to be called again, and `firstRender` will then be set to true.

Using `React.StrictMode` renders components twice, and the way we handle mounting causes two charts to be mounted. The latest change will prevent that, which I assume is what the rendering promise accomplished.